test/action_controller/json/include_test.rb

```diff
@@ -158,8 +224,14 @@ def test_render_resource_with_deep_include
               ]
             }
           }
+        end
 
-          assert_equal(expected, response)
+        def with_default_includes(include_tree)
+          original = ActiveModel::Serializer.config.default_includes
```

Comment:
should be `ActiveModelSerializers.config`

also this has no effect on the memoized default, right?